docs/root/start/install.rst

```diff
@@ -57,38 +57,16 @@ using `Get Envoy <https://www.getenvoy.io/>`__.
    To add the nightly repository instead, replace the word ``stable`` with ``nightly``,
    when adding the ``apt`` repository.
 
-Install Envoy on CentOS Linux
+Install Envoy on RPM-based distros: CentOS/RHEL/Suse/Amazon Linux/Fedora etc
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
underline needs to match the length of the title - can you fix @ankatare and ill review the generated docs - thanks

Reply:
Sure. doing it. some issues in my local env so could not test it locally.

Reply:
it again failed :( . let me work on my local env first ...

Reply:
same problem - underline length needs to match title in rst (it can actually be longer, but please dont do that!)

Reply:
I hope it is done now